File: Beginner/1036 (Bhaskara's Formula)/1036.py
a,b,c = input().split()

a = float(a)
b = float(b)
c = float(c)

delta = float((b * b) -4 * (a * c))

if a <=0 or delta < 0:
    print("Impossivel calcular")
else:
    print("R1 =","{:.5f}".format(((-b) + (delta **0.5))/(2*a)))
    print("R2 =","{:.5f}".format(((-b) - (delta **0.5))/(2*a)))

File: Beginner/1036 (Bhaskara's Formula)/1036.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] texto = Console.ReadLine().Split(' ');
            double a= double.Parse(texto[0]), b = double.Parse(texto[1]), c = double.Parse(texto[2]);
            double result;
            double delta = (b * b) - 4 * (a * c);
            if(a <=0 || delta < 0)
            {
                Console.WriteLine("Impossivel calcular");
            }
            else
            {
                result = ((-b) + Math.Sqrt(delta)) / (2 * a);
                Console.WriteLine("R1 = " + string.Format("{0:0.00000}",result));
                result = ((-b) - Math.Sqrt(delta)) / (2 * a);
                Console.WriteLine("R2 = " + string.Format("{0:0.00000}", result));
            }
        }
    }
}